survey.tex

```diff
@@ -2,6 +2,11 @@
 % Responsible : Zeljko
 \section{Survey Cadence}
 
-Early Science observations should align as closely as possible with main survey and ultimate long-term science goals
-Details on the implications for the Survey Cadence of the \esp will be added in future as the outcome of commissioning becomes clear.  
+% Early Science observations should align as closely as possible with main survey and ultimate long-term science goals
+% Details on the implications for the Survey Cadence of the \esp will be added in future as the outcome of commissioning becomes clear.
 
+The early science observations can be separated into two kinds: commissioning observations, and survey observations taken during the first 6 months of the LSST.
+Only the latter---the ``early survey observations`` will be included in the LSST data processing.
```

Comment:
This surprises me. I would have thought that there might be some motivation to reprocess the commissioning-era data. I expect that those could have significant legacy value because of the depth and dense temporal sampling that would be useful to bootstrap the main LSST survey. Presumably the Science Pipelines will be evolving and having a consistent processing would be valuable.

Reply:
Agreed.   All data taken to date passing a quality cut

Reply:
Phil wrote this and I had not reviewed it. @drphilmarshall  - what were your thoughts here?

Reply:
I had understood the LSST to be a self-contained 10-year survey, with cadence designed so as to provide uniform sky coverage after 10 years (and also in at least some intermediate years), and _that does not include any observations taken during the commissioning period._ For example, I don't think any of the simulated LSST cadences include the SV survey visits. So, I don't think the Y1 cadence needs to be adjusted to accommodate what was taken in commissioning (eg so as to achieve uniform sky coverage at the end of Y1 once the SV images are included in the coadds). So, I wasn't saying that the SV images won't be re-processed - just that they would not be included in the LSST survey.

I don't know what is planned regarding periodic re-processing of the SV images during the LSST but Keith's point about their legacy value is a good one - perhaps it does make sense to keep refining the DP2 dataset. 

So, what to write? I'll remove my categorical statement, not least because it seems to be confusing. Perhaps we can get by without this sentence at all - I'll take a look.

Reply:
I just pushed a commit to remove the confusing sentence. Simplified text now looks like this:
<img width="702" alt="image" src="https://user-images.githubusercontent.com/710903/197141068-444051a6-323c-4acf-b43d-6abe9a940f39.png">
Hope this looks OK now.

Reply:
Thanks Phil, that is clearer and I understand what you mean now.  It was the "will be included in the LSST data processing." that was confusing.  The commissioning data will be processed by the LSST science pipelines (probably more than once)  but will not form part of the formal 10-year dataset.

Reply:
@drphilmarshall  I'd suggest to be less specific - e.g 
"Optimizing the LSST year 1 observing schedule to maximize early science may mean that the time sampling in the LSST survey region will look somewhat different to that of subsequent years. "

We don't know that it will look different, but it may.